provider/cloudflare/cloudflare_test.go

```diff
@@ -1801,7 +1791,7 @@ func TestCloudflareDNSRecordsOperationsFail(t *testing.T) {
 			shouldFail: true,
 		},
 		{
-			Name: "failing to list DNS record",
+			Name: "adding failing to list DNS record",
```

Comment:
```suggestion
			Name: "adding to list DNS record",
```

Reply:
same as for the custom hostname, first we add "failing to list" DNS record, then the next change (any, empty would do) is causing to attempt listing the added one and expected to fail

Reply:
Ok, I took the time to re-read this test and run it locally. I get the logic now. Thanks.